src/components/ChatRoom/ChatRoom.tsx

```diff
@@ -43,6 +43,11 @@ export const ChatRoom = () => {
 		if (messagePhoto) {
 			messageDoc.messageImageUrl = messagePhoto;
 		}
+
+		const input = formValue.replace(/  +/g, ' ');
```

Comment:
You only need boolean value. Change the name to something like `const isTextValid` and replace it to just checking if the text matches the regex.